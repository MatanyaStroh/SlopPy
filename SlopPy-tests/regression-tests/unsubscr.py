x = 5[10]
assert type(x) is NA
