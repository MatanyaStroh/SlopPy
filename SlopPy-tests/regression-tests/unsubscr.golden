{NA <type 'exceptions.TypeError'> "'int' object is unsubscriptable"
 Traceback (most recent call last):
  File "regression-tests/unsubscr.py", line 1, in <module>
    x = 5[10]

Locals: {'__builtins__': <module '__builtin__' (built-in)>, '__name__': '__main__', '__file__': 'regression-tests/unsubscr.py', '__doc__': None, '__package__': None}
}
